### PR TITLE
Add dependencies and configuration files

### PR DESCRIPTION
**Pull request by TimKDJ**

### CODE REVIEW

File: R/emmeans.rma.R [hunk 1]
```diff
@@ -115,96 +115,3 @@ addSignifStars <- function(x, ..., column = "P(>|t|)", stars = c(' ***'=.001, '
...
-  M = do.call(model.matrix, list(mdlformula, tmp)) #, contrasts = list(wool='contr.Treatment', tension='contr.Treatment')))
-  Kmf = model.frame(paste("~ 0 + ",all.vars(formula), collapse = ":"), unique(tmp[, all.vars(formula), drop = FALSE]))
-  K = model.matrix(Kmf, tmp, contrasts = as.list(setNames(rep("contr.Treatment", length(all.vars(formula))), all.vars(formula))))
-  K.M = ginv(K) %*% M
-  em = K.M %*% b
-  se = diag(K.M %*% Vb %*% t(K.M))^.5
-  ret = transform(Kmf, emmean = em, SE = se, df = df.residual(fit))
-  structure(transform(ret, lower.CL = emmean - qt(.025, df)*se, upper.CL = emmean - qt(0.5-conf.level/2, df)*se), 
-            collapsed = setdiff(all.vars(mdlformula), all.vars(formula)), conf.level = conf.level)
-})#() # () # for testing the correctness
```
Comment (TimKDJ) [inline R/emmeans.rma.R:142]: Any reason in particular we had this @FBartos?

Comment (FBartos) [reply R/emmeans.rma.R:142]: no idea, this part was probably implemented by @sophieberkhout / Raul?

Comment (sophieberkhout) [reply R/emmeans.rma.R:142]: I don't know what this is.

Comment (TimKDJ) [reply R/emmeans.rma.R:142]: Ah, yes, this seems to have been around for a bit.

**Pull request merged**